src/reconstruct/helper.ts

```diff
@@ -42,13 +42,19 @@ export function merge(snipbits: Code[]): Code {
 	for(const code of snipbits) {
 		for(const line of code) {
 			for(const part of line.linePart) {
-				buckets[part.loc.line].linePart.push(part)
+				const lineNumber = part.loc.line
+				if(buckets[lineNumber] === undefined) {	//may be necessary as empty elements count as undefined and we don't want to reassign filled buckets
+					buckets[lineNumber] = {linePart: [], indent: line.indent}
+				}
+				buckets[lineNumber].linePart.push(part)
 			}
 		}
 	}
 
 	//sort buckets by column and stich lines into single code piece
 	for(const line of buckets) {
+			continue
+		}
```

Comment:
Ist die Einrückung so gedacht? also die Schleife sollte so doch tot sein :eyes:

Reply:
da fehlt evtl ein if-statement aber das problem das hier versucht wird zu lösen ist, buckets ja empty spaces hat und die sind undefined. Hier versuche ich höchstwahrscheinlich gerade diese empty spaces zu skippen, aber ich schau mir das im code nochmal genauer an.

Reply:
der code ist evtl in der ide anderst geformated, aber continue sollte doch nur zum nächsten schleifen durchlauf springen?

Reply:
ja, aber da da ja kein if davor kommt, geht das einmal als "no-op" einfach über alle buckets - oder blendet github das if nicht ein? :D

Reply:
Also im full view der änderungen ist ein if dabei, und ich bin mir auch recht sicher, dass ich eins drin hatte😅

Reply:
Spannend, dass das die diff view  so unterschlägt.